### ISSUE TITLE
garden ordnance survey does not work if using garden in BPM

### ISSUE THREAD

**Issue by sunjayBhatia** [cloudfoundry/garden-runc-release#118]
Running the garden ordnance survey (`curl bit.ly/garden-ordnance-survey -sSfL | bash`) on a VM where garden is run in BPM fails to generate a full report:

```
diego-cell/3e5d04fe-d2bd-408b-a9dc-01f56a66f869:/tmp# curl bit.ly/garden-ordnance-survey -sSfL | bash
find: ‘/var/vcap/data/garden/depot’: No such file or directory
```

**Comment by julz** [comment]
Hi @sunjayBhatia - `ordnance-survey` isn't supported in bpm, but we now ship a `/var/vcap/packages/dontpanic/bin/dontpanic` binary... which may also not work in bpm :-). Converting this in to a feature to ensure `dontpanic` _does_ work with bpm enabled..

**Comment by Callisto13** [comment]
fixed in GRR 1.18.3
